schema/compose-spec.json

```diff
@@ -588,6 +588,22 @@
       }
     },
 
+    "require": {
+      "id": "#/definitions/require",
+      "oneOf": [
+        {"type": "string"},
+        {
+          "type": "object",
```

Comment:
I think this needs to be `"type": "array"` and then this definition would be for `"items"`

Reply:
yes indeed, this is incomplete copy/paste from https://github.com/compose-spec/compose-go/pull/416/files#diff-5f5b95e68cadf606dc6b4956ca67b87039e3283cb01aa3208fda399abc167906